docs/_lab/09-writing-advanced-plans/index.md

```diff
@@ -61,28 +61,28 @@ Finished: command 'uptime' with 0 failures in 0.39 sec
 Plan completed successfully with no result
 ```
 
-**Note:** Running the plan multiple times results in different output. As the return value of the task is random, the command runs on a different subset of nodes each time.
+Running the plan multiple times results in different output. As the return value of the task is random, the command runs on a different subset of nodes each time.
 
-# Write a plan with custom Ruby functions
+## Write a Plan With Custom Ruby Functions
 
 Bolt supports a powerful extension mechanism via Puppet functions. These are functions written in Puppet or Ruby that are accessible from within plans, and are in fact how many Bolt features are implemented. You can declare Puppet functions within a module and use them in your plans. Many existing Puppet functions, such as `length` from [puppetlabs-stdlib], can be used in plans.
 
-Save the following as `modules/exercise9/plans/count_volumes.pp`:
+Save the following as `Boltdir/site-modules/exercise9/plans/count_volumes.pp`:
 
 ```puppet
-{% include_relative modules/exercise9/plans/count_volumes.pp -%}
+{% include lesson1-10/Boltdir/site-modules/exercise9/plans/count_volumes.pp -%}
 ```
 
-To use the `length` function, which accepts a `String` type so it can be invoked directly on a string, install it locally:
+To use the `length` function, which accepts a `String` type so it can be invoked directly on a string, install it locally in the `modules` directory:
 
 ```bash
-git clone https://github.com/puppetlabs/puppetlabs-stdlib ./modules/stdlib
+git clone https://github.com/puppetlabs/puppetlabs-stdlib ./Boltdir/modules/stdlib
 ```
 
 Run the plan.
 
 ```bash
-bolt plan run exercise9::count_volumes nodes=all --modulepath ./modules
```

Comment:
Same

Reply:
exercise9::count_volumes fails on the Windows node because it does not recognize the `df` command which is part of the plan.